src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -3650,29 +3646,27 @@ import javax.annotation.Generated;
 @Generated("by GAPIC")
 @BetaApi
 public class FakebillingClient implements AutoCloseable {
-  private final FakebillingSettings settings;
+  private final ProjectAdminSettings settings;
```

Comment:
The class name should be ProjectAdminClient (interface config from yaml) instead of FakeBillingClient (api name), but that hasn't been implemented yet.